test/test_unistd.rs

```diff
@@ -669,9 +676,10 @@ fn test_fpathconf_limited() {
     );
 }
 
+#[cfg(not(target_os = "hurd"))]
```

Comment:
What about this and the one in line 692, we should probably change them to `#[cfg_attr(target_os = "hurd", ignore)]`

Reply:
ok, done so